### PR TITLE
feat(ui): Add mobile app settings

### PR DESCRIPTION
**Pull request by matejminar**
Relates to: https://github.com/getsentry/sentry-mobile/pull/118

https://user-images.githubusercontent.com/9060071/108514689-75c60400-72c4-11eb-880f-43affc739acb.mp4

This is feature flagged work in progress.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.tsx [hunk 1]
```diff
@@ -133,6 +133,11 @@ const SidebarDropdown = ({api, org, orientation, collapsed, config, user}: Props
                     <SidebarMenuItem to="/settings/account/api/">
                       {t('API keys')}
                     </SidebarMenuItem>
+                    {user.isSuperuser && (
+                      <SidebarMenuItem to="/settings/account/api/mobile-app/">
+                        {t('Mobile app')}
```
Comment (HazAT) [inline src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.tsx:138]:
```suggestion
                        {t('Mobile App')}
```

**Review by HazAT — APPROVED** [review]
One nit, other lgtm

File: src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.tsx:138]: This is for the top left menu where other items are not in title case:
image
Should we still rename it to `Mobile App`?

File: package.json [hunk 2]
```diff
@@ -102,6 +102,7 @@
     "po-catalog-loader": "2.0.0",
     "prism-sentry": "^1.0.2",
     "prop-types": "^15.6.0",
+    "qrcode.react": "^1.0.1",
```
Comment (EvanPurkhiser) [inline package.json:105]: Can we replace https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/qrcode.tsx with this?

Comment (matejminar) [reply package.json:105]: @EvanPurkhiser I was considering that component but it looked like it only supports some sort of matrix data as a prop from authenticator - does it work with regular strings?

**Pull request merged**